### PR TITLE
Updated README.md with some style 💕

### LINKED ISSUE #2: Make a PR that updates the readme

### PR DESCRIPTION
**Pull request by flaally**
close #2

### CODE REVIEW

**Review by brunogrcsada — APPROVED** [review]
This looks great! We've made it :)

**Review by cazumara — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
-# bmw-objects-test
\ No newline at end of file
+<h1 align="center">
+  <br>
+  <a href="https://github.com/missKatiaPunter/bmw-objects-test"><img src="https://lh3.googleusercontent.com/proxy/QHRaBSHEFFFYyZb0xX842N5RJXWdoxg_CFmUMyPDPMs4i8-cQVh8zxdFzDlukl_H_hrYwOcq3VS94VwG7tx8NOa7BJiA5tR1KASV5aCsG918" alt="BMW Logo" width="200"></a>
+  <br>
+  BMW Objects Test
+  <br>
+</h1>
```
Comment (cazumara) [inline README.md:7]: lalalala, good code.

Comment (cazumara) [reply README.md:7]: @flaally lalalalala

**Review by missKatiaPunter — CHANGES_REQUESTED** [review]
@cazumara and @flaally an you delete line 20

File: README.md (hunk 1 above)
Comment (cazumara) [reply README.md:7]: @flaally isdiubnsdc

**Review by missKatiaPunter — APPROVED** [review]

**Pull request merged**